packages/azure-http-specs/specs/azure/resource-manager/common-properties/mockapi.ts

```diff
@@ -88,6 +95,11 @@ Scenarios.Azure_ResourceManager_CommonProperties_ManagedIdentity_createWithSyste
     method: "put",
     request: {
       body: json({
+        location: "eastus",
+        tags: {
+          tagKey1: "tagValue1",
+        },
+        properties: {},
```

Comment:
The empty `{}` is what Java needs?

Reply:
I think Java sends the empty object in `properties` property.

The resource has a model for `properties`. Just it only contain read-only property.
https://github.com/Azure/typespec-azure/blob/a1ab10e20e53d4738f196d0df539bba01d842083/packages/azure-http-specs/specs/azure/resource-manager/common-properties/managed-identity.tsp#L18-L36

Reply:
If user don't set the properties, shouldn't we send nothing?

Reply:
I tried to find out a workaround in .net to send the `{}`, but failed.

Reply:
Sending a `{}` and sending nothing is not exactly same...

Would Dapeng know about how this be sent? I assume ARM resource would send a `properties` in almost all cases. Just in this case there is no writable property in this model.

Reply:
Our logic is if you don't touch the properties, then send nothing. Or maybe you can have a value set for `Properties.ProvisioningState` so that we can touch it?

Reply:
Yeah, I can add a writable property within `properties`. But it would require test change in emitters (as they now need to set this).

So here I just add a `handler` to again only verify the `identity` whether this test is for.